Allow reading COMMANDLINE_ARGS from file

GenericPseudonym opened the pull request:
To avoid having custom settings clobbered by merge/pull/etc.

Commit:
Allow reading COMMANDLINE_ARGS from file

To avoid having custom settings clobbered by merge/pull/etc.

AUTOMATIC1111 commented:
Does pulling do that? It's supposed to merge whatever you changed with incoming changes.

Commit:
Merge branch 'master' of https://github.com/AUTOMATIC1111/stable-diffusion-webui

Commit:
Allow passing -1 as a seed for xy_grid

Don't check for "-" at the beginning or end of strings,
since that wouldn't be a valid "1-10" type numerical range anyways.

dgrenner commented:
Pulling usually gets blocked:
```
git pull
error: Your local changes to the following files would be overwritten by merge:
        webui.bat
Please commit your changes or stash them before you merge.
Aborting
```
It can of course be handled by stash / pull / apply stash. But was thinking about something similar myself as to this PR and #140, which is trying to work around the same problem in a different way. But would be good to be able to overwrite the other settings as well (I need the PYTHON path configured as well).

I suggest adding:
```
if exist webui.settings.bat (
    call webui.settings.bat
)
```
to line 8 on webui.bat - then users can create that file with whatever overwrites they want. And then add webui.settings.bat to .gitignore. Would need a readme update as well, instructing users to add whatever they want in there. If there are more settings in webui.bat it should be added before, so that also could be modified. If you want @AUTOMATIC1111 I can create a separate PR for that (instead of hijacking this one - sorry for that).

AUTOMATIC1111 commented:
I merged the other solution adding a more customizable config file.
